app/containers/CustomHeader/components/HeaderTitle/index.tsx

```diff
@@ -18,24 +18,14 @@ const HeaderTitle = ({ headerTitle }: IHeaderTitle) => {
 	if (typeof headerTitle === 'string') {
 		if (isAndroid) {
 			return (
-				<Text
-					numberOfLines={1}
-					style={{
-						...styles.androidTitle,
-						color: colors.fontTitlesLabels
-					}}>
+				<Text numberOfLines={1} style={[style || styles.androidTitle, { color: colors.fontTitlesLabels }]}>
```

Comment:
You're insisting on this pattern, so let me tell you why it doesn't work.
```jsx
androidTitle: {
	...sharedStyles.textBold,
   fontSize: 18,
	flex: 1,
	lineHeight: 24,
	paddingVertical: 10
}
...
style || styles.androidTitle
```
This is going to use style for any styles it may have, but you can't predict which styles were sent as props.
What if the only thing I want to change on the style is `paddingVertical: 8`?

That's why we inherit style like this.
```suggestion
				<Text numberOfLines={1} style={[styles.androidTitle, { color: colors.fontTitlesLabels }, style]}>
```